readthedocsext/theme/templates/includes/elements/promotions/pull-request-previews.html

```diff
@@ -0,0 +1,10 @@
+<div class="ui card">
+  <div class="content">
+    <a class="header" target="_blank" href="https://docs.readthedocs.io/page/pull-requests.html">Preview changes before deploying!</a>
+    <div class="description">
+      Easily catch documentation formatting and display issues with Pull Request Previews.
+      Stop worrying about having an up-to-date local build environment to build the docs locally,
+      and always have a sharable link with production-ready documentation before going live.
+    </div>
```

Comment:
I feel like these should probably have a call to action button. The linked header might not be obvious and we're not using actionable language in the headers here either.

This might be too much for a button, so maybe just a link, but:

```suggestion
      <a class="ui small basic button">{% trans "Learn about pull request previews" %}</a>.
    </div>
```

If just a link looks best, you probably also want to wrap the above text blocks in `<p>`.